src/core/qgsapplication.cpp

```diff
@@ -623,6 +623,11 @@ QString QgsApplication::i18nPath()
     return ABISYM( mPkgDataPath ) + QStringLiteral( "/i18n/" );
 }
 
+QString QgsApplication::metadataPath()
```

Comment:
const

Reply:
This one is `static` in the header already